metadata/metadata.go

```diff
@@ -162,6 +162,23 @@ func HasLabelWithValue(obj v1.Object, key, value string) bool {
 	return ok && val == value
 }
 
+// SetAnnotation creates the annotation or updates the existing annotation.
+func SetAnnotation(obj v1.Object, key string, value string) error {
```

Comment:
Could you add also a function for labels?

Reply:
added in new commit